FSW/src/headers/config.hpp

```diff
@@ -114,13 +110,13 @@ const int WD_PULSE_DUR_MICROSEC = 10;       // microseconds, watchdog reset sign
 const int HK_SAMPLES_TO_KEEP = 5000;   // number of previous housekeeping samples to keep
 
 // heater cutoff temperatures
-static bool FORCE_HEATER_ON = false;   // if true, heater will always be on regardless of temperature
+static volatile bool FORCE_HEATER_ON = false;   // if true, heater will always be on regardless of temperature

```

Comment:
declaring them as `volatile` makes the compiler stop yelling at us that they are unused, but since you discovered that all the static variables in `config` are useless, this is probably irrelevant and will be changed.